xml/art-obs-beginners-guide.xml

```diff
@@ -1076,11 +1161,14 @@ Patch0:         &gitproject;_main.diff</screen>
      <para>
       The changes from user <systemitem class="username"
        >&obsuser2;</systemitem> are not integrated into the
-      <uri>&obshome1;</uri> project. For example, the reasons to decline a
-      submit request are usually build errors, style issues, or other
-      problems. Usually, the declined submit request contains a message which
-      contains the reason. Corrections has to be done in a new submit request
-      by superseding the old one.
+      <uri>&obshome1;</uri> project.
+     </para>
+     <para>
+      For example, reasons to decline a
+      submit request are build errors, style issues.
+      When declining submit request, the reviewer will usually give a
+      reason. &obsuser2; can then correct their submission using a new submit
+      request that supersedes the previous one.
```

Comment:
maybe point out also that requests usually submit a concrete version of the source, later changes don't get fetched by default. This is unlike github pull requests where you can modify also afterwards with the risk the a required review step might get missed.